libraries/AP_HAL_ChibiOS/hwdef/MFT-SEMA100/hwdef.dat

```diff
@@ -0,0 +1,240 @@
+# hw definition file for processing by chibios_hwdef.py for MFT-SEMA100
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+# board ID for firmware load
+APJ_BOARD_ID 1500
+
+FLASH_SIZE_KB 2048
+
+# with 2M flash we can afford to optimize for speed
+env OPTIMIZE -O2
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 2
+
+
+
+USB_STRING_MANUFACTURER "MFT"
+USB_STRING_PRODUCT "MFT-SEMA100"
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 128
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART1 USART3 UART5 UART7 UART8
```

Comment:
adding OTG2 at the end to give dual USB endpoints is normally worthwhile, helpful for CAN diagnostics